skyportal/handlers/api/source.py

```diff
@@ -654,6 +654,8 @@ class Validator(Schema):
                     else None
                 )
 
+            source_info["color_magnitude"] = get_color_mag(source_info["annotations"])
```

Comment:
Will this be widely relevant for most API users? If not, we should make this optional via a query param that defaults to false

Reply:
I thought about this, but then you will want to have it true for all the source tables. The idea is that this is how you know if the source is transient or not.